content/en/continuous_integration/guides/add_custom_metrics.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Add Custom Metrics to your Tests
+kind: guide
+description: Learn how to use custom metrics (measures) in your tests
+further_reading:
+  - link: "/continuous_integration/tests"
+    tag: "Documentation"
+    text: "Exploring tests"
+---
+
+
+{{< site-region region="gov" >}}
+<div class="alert alert-warning">CI Visibility is not available in the selected site ({{< region-param key="dd_site_name" >}}) at this time.</div>
+{{< /site-region >}}
+
+<div class="alert alert-warning"><strong>Note</strong>: These custom metrics are <strong>not</strong> <a href="/metrics">Datadog Metrics</a>. They are numerical tags (also known as measures) that represent things like memory usage or request rates.</div>
+
+This guide will walk you through adding and using custom metrics for your tests.
+
+### Add the custom metric to your test
+
+Before you can follow this guide, make sure that [Test Visibility][1] is already setup for your language.
+
+First step is to add the custom metric to your test. The native instrumentation allows you to use the programmatic API:
+
+{{< tabs >}}
+{{% tab "Javascript/Typescript" %}}
+```javascript
+  it('sum function can sum', () => {
+    const testSpan = require('dd-trace').scope().active()
+    testSpan.setTag('test.memory.rss', process.memoryUsage().rss)
+    // test continues normally
+    // ...
+  })
+```
+{{% /tab %}}
+
+{{% tab "Java" %}}
+```java
+// inside your test
+final Span span = GlobalTracer.get().activeSpan();
+if (span != null) {
+  span.setTag("test.memory.usage", 1e8);
+}
+// test continues normally
+// ...
+```
+{{% /tab %}}
+
+{{% tab "Python" %}}
+```python
+from ddtrace import tracer
+import os, psutil
+
+# Declare `ddspan` as argument to your test
+def test_simple_case(ddspan):
+    # Set your tags
+    process = psutil.Process()
+    ddspan.set_tag("test.memory.rss", process.memory_info().rss)
+    # test continues normally
+    # ...
+```
+{{% /tab %}}
+
+{{% tab ".NET" %}}
+```csharp
+// inside your test
+var scope = Tracer.Instance.ActiveScope; // from Datadog.Trace;
```

Comment:
there's a `using` we need at the top of this, right @tonyredondo ?

Reply:
the extension method is declared inside `Datadog.Trace` as `Tracer` so we don't need the using statement.